src/components/mgt-login/mgt-login.ts

```diff
@@ -120,13 +132,11 @@ export class MgtLogin extends MgtBaseComponent {
 
   firstUpdated() {
     window.addEventListener('click', (event: MouseEvent) => {
-      if (event.target !== this) {
-        // get popup bounds
-        const popup = this.renderRoot.querySelector('.popup');
-        if (popup) {
-          this._popupRect = popup.getBoundingClientRect();
-          this._showMenu = false;
-        }
+      // get popup bounds
```

Comment:
Curious why this change?

Reply:
This was to fix issues with retemplating originally and clicking to get menu to appear.  The original fix was to remove this check and add a `event.stopPropagation();` below in one of the if conditions.

Someone as part of some other change added the stopPropagation below to all cases, but didn't remove this condition.